content/en/serverless/troubleshooting/installing_the_layer.md

```diff
@@ -136,7 +136,7 @@ Globals:
 [AWS SAM docs][2]
 
 [1]: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#globals-section
-[2]: https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/what-is-sam.html
+[2]: https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/what-is-sam.html-->
```

Comment:
This content should not appear on this page, so I commented it out for now. I will give it a refresh later this week.